source/styleguide/molecules/SgPageShell/SgPageShell.css

```diff
@@ -86,17 +78,3 @@
     }
   }
 }
-
-.SgScrollbars {
```

Comment:
Lets leave the custom scroll bars please. This is for Windows users like me who don't get nice fancy thin OSX scroll bars :D